app/javascript/app/components/ghg-emissions/ghg-emissions-component.jsx

```diff
@@ -97,11 +97,22 @@ function GhgEmissions(props) {
       onClick: handleInfoClick
     },
     {
-      type: 'download-combo',
-      section: 'ghg-emissions',
-      link: downloadLink,
-      tooltipText: 'View or download raw data',
-      positionRight: true
+      type: 'downloadCombo',
+      options: [
+        {
+          label: 'Download current data (CSV)',
+          action: handleDownloadDataClick
+        },
+        {
+          label: 'Save as image (PNG)',
+          action: () => {}
+        },
+        {
+          label: 'Go to data explorer',
+          link: downloadLink
```

Comment:
This is opening a new tab and probably we dont want to do that as it is internal navigation. What do you think @faustoperez ?